### DISCUSSION TITLE
Esp32-c3 WIZ chip in WIZ bulb istead of esp32-solo

### DISCUSSION THREAD

**Discussion by kroon040** [arendst/Tasmota#20484] [Q&A]
Hi 

I bought this device, because i wanted a buld with esp32 chip

https://templates.blakadder.com/wiz_9290024502.html

I did open the bulb, same partnumber , and what I see, and es32-c3. Only I can't find the the right pins. I hope that someone knows a datasheet for it but it looks like an own WIZ esp32-c3. It has more pins 8 in total to the print. I can see op the print the tx, rx and gnd but not de GPIO8 and 9 for flashing it. It look like the ESP8684-WROOM-07 ESP32-C2 CBLC5, but this one have less pins to the print and not the full shape.  I hoped that there is a replacement for ...

![IMG_3817 - 1mEinT54hB43x59LCSRYsQyQmdtPbAqnD](https://github.com/arendst/Tasmota/assets/40452265/28d1d0ab-4632-4f5e-8d4f-a94f495e6f2e)
![IMG_3818](https://github.com/arendst/Tasmota/assets/40452265/77ee0c8e-b618-4bde-8198-23cf04656ba9)

**Comment by lalo-uy** [comment]
In the picture 2 more test point can be seen. Have you checked where are
them connected, visually or with a continuity tester?

**Comment by kroon040** [comment]
The TX an RX are on the chip, but not connected to the printboard of the bulb, only the GND I could find

I alo have another photo

![Esp32-c3](https://github.com/arendst/Tasmota/assets/40452265/28293786-4d5f-487e-8fc8-5ce3d2630cad)

**Reply by sfromis** [reply]
Looks like https://fccid.io/2AGBW-WIZ2012/

And I'd guess that the "Boot" pad is is gpio 9.

To do anything, you need to use those small pads, do not expect the connections to the main PCB to be useful for flashing, outside of having power/ground available.

From what I can see in the FCC photos, you have vcc on the opposite side from the gnd you already located. 
![image](https://github.com/arendst/Tasmota/assets/47082390/5c3453c7-bc87-41f5-a1b3-c21697013d32)

**Comment by kroon040** [comment]
Looks like the chip can't be programmed encrypted , and there is I think also no replace module

https://github.com/arendst/Tasmota/issues/3055#issuecomment-1890472448
